### PR TITLE
Allows wallpapers to be set without external hash file

### LINKED ISSUE #23: Allow wallpapers without Hash
Sometimes is not possible to use the hash for a file. Let's enable to download wallpapers without the currently required hash. Then this can be enforces when the directives server automatically calculate the hash for the uploaded wallpapers.

### PR DESCRIPTION
**Pull request by DT3264**
With this change the hash will not be an external file
Rather it would be stored in the variable WallHash (keeping that name atm since using WallpaperHash gives issues with the wallpaper url, unless Wallpaper gets renamed to WallpaperUrl)
Example: `WallHash=c402cccda9d3a63058513c3169ee75908dca49b9227047b69e38f86270a91508  bts.jpg`

Closes #23

### CODE REVIEW

File: base-system/usrroot/usr/lib/hos/utils.sh [hunk 1]
```diff
@@ -34,24 +34,18 @@ safe_download(){
...
-		pushd "$TMP"
-		if ! sha256sum --status --check "$TMP_HASH_FILE"; then
-			echo "Downloaded file does not match it's hash, aborting. $FILE_URL" 1>&2
-			popd
-			rm -rf "$TMP"
-			return 1 # Error
-		fi
+	pushd "$TMP"
+	## Write hash into hash file
+	echo "$FILE_HASH" > "$TMP_HASH_FILE"
```
Comment (equetzal) [inline base-system/usrroot/usr/lib/hos/utils.sh:39]: Why not to ask for the single hash and append the file name? 
Or do you see some advantage from using also the filename? (eg. more security when changing wallpaper?)

File: base-system/usrroot/usr/sbin/hos-wallpaper [hunk 2]
```diff
@@ -66,7 +66,7 @@ prepare_wallpaper(){
 	SECTION="$1"
 	CURRENT_MODE="$2"
 	WALLPAPER_URL="$(hos-dvar --variable-name Wallpaper --section "$SECTION")"
-	WALLPAPER_HASH_URL="${WALLPAPER_URL}.sha256"
+	WALLPAPER_HASH="$(hos-dvar --variable-name WallHash --section "$SECTION")"
```
Comment (equetzal) [inline base-system/usrroot/usr/sbin/hos-wallpaper:69]: With the new PR #67 I definitely preffer the explicit name now. 'WallpaperHash' or 'WalpaperSha256' sounds good.

**Review by equetzal — APPROVED** [review]
Overall good!!, I think that maybe this PR is good moment for updating the docs too about the walpaper as this will differ from the original directives spec.
@RafagaBlanca to please skip this spec or sync with Daniel to include this changes on the docs.

File: base-system/usrroot/usr/lib/hos/utils.sh (hunk 1 above)
Comment (DT3264) [reply base-system/usrroot/usr/lib/hos/utils.sh:39]: Actually that's a good idea leaving the hash only, I didn't though on that possibility :p

File: base-system/usrroot/usr/sbin/hos-wallpaper (hunk 2 above)
Comment (DT3264) [reply base-system/usrroot/usr/sbin/hos-wallpaper:69]: I think WallpaperSha256 is explicit enough

**Pull request merged**